### DISCUSSION TITLE
Is it possible to attach an Avalonia Window object to an NSWindow pointer?

### DISCUSSION THREAD

**Discussion by baochenw** [AvaloniaUI/Avalonia#15555] [Q&A] [answered]
Can an Avalonia Window object be attached to an NSWindow pointer to use Avalonia's API for window-related operations? For example, I have a pointer to an NSWindow created in Objective-C, and I want to use an Avalonia Window to create a modal dialog on top of it.

**Comment by maxkatz6** [comment, marked as answer]
No, it's not possible currently.
Embedding into a NSView is technically possible, but wasn't implemented.
